src/__tests__/fireEvent.test.js

```diff
@@ -78,18 +80,6 @@ describe('fireEvent', () => {
 
     expect(handlerMock).toHaveBeenCalledWith(EVENT_DATA);
   });
-
-  test('should not bubble event to root element', () => {
-    const onPressMock = jest.fn();
-    const { getByTestId } = render(
-      <TouchableOpacity onPress={onPressMock}>
-        <Text testID="test">Content</Text>
-      </TouchableOpacity>
-    );
-
-    expect(() => fireEvent.press(getByTestId('test'))).toThrow();
-    expect(onPressMock).not.toHaveBeenCalled();
-  });
```

Comment:
This test seems fishy and not testing what it's supposed to. Not really sure why it didn't fail earlier, since the `test` testID is there, we can press it, and closest `onPress` is bound, so it should be called.